src/router.ts

```diff
@@ -215,29 +230,40 @@ class Router {
   async add_dip(
     expirationSec: number,
     strike: number,
-    splMint: PublicKey,
+    baseMint: PublicKey,
+    quoteMint: PublicKey,
+    callOrPut: CallOrPut,
     connection: Connection,
   ): Promise<void> {
     const [optionMint] = await findProgramAddressWithMintAndStrikeAndExpiration(
       OPTION_MINT_ADDRESS_SEED,
-      strike * NUM_DIP_ATOMS_PER_TOKEN,
+      strike,
```

Comment:
why removing num_dip_atoms_per_token?

Reply:
L309 changed to strikeAtomsPerToken

Reply:
Allows the flipping to be cleaner in L250-253

Reply:
Change the name of the variable to be clear that it is atoms